### PR TITLE
Disabling parallel execution of jitted functions

### PR DESCRIPTION
**Pull request by herbertludowieg**
Should take care of updating the code to be compatible with numba=0.48
- Changed `nbpll` variable to always be false in `exatomic.base`
- Changed any instances where we have `parallel=True` to `parallel=nbpll` for the jitted functions

Used the following code to ensure that we can open a UniverseWidget scene and plot the orbitals
``` python
from exatomic import gaussian
from exatomic.base import resource
from exatomic import UniverseWidget as UW

fni = gaussian.Fchk(resource('g09-ch3nh2-631g.fchk')).to_universe()

fni.add_molecular_orbitals(vector=range(10))

UW(fni)
```

### CODE REVIEW

File: exatomic/interfaces/tests/test_cube.py [hunk 1]
```diff
@@ -52,5 +53,6 @@ def test_uni_from_cubes_rotate_and_write(self):
         self.assertEqual(len(self.uni.field.field_values), 2)
         rot = self.uni.field.rotate(0, 1, np.pi / 4)
         self.assertEqual(rot.shape[0], 2)
-        f = Cube.from_universe(self.uni, 1)
-        self.assertEqual(len(f), 874)
+        if not (platform == 'win32' or platform == 'cygwin'):
```
Comment (herbertludowieg) [inline exatomic/interfaces/tests/test_cube.py]: Rather not do this, but it seems to be simplest workaround for the max_length error in windows

File: appveyor.yml [hunk 2]
```diff
@@ -56,9 +56,10 @@ install:
   - cmd: conda create -n test python=%CONDA_PY%
   - cmd: activate test
   - cmd: conda update -q -y --all
-  - cmd: conda install -q -y -c conda-forge numpy scipy seaborn networkx pandas
-  - cmd: conda install -q -y -c conda-forge pytest pytables six numba='<0.47' sympy pyyaml
-  - cmd: conda install -q -y -c conda-forge psutil numexpr nodejs ipywidgets bokeh
+  - cmd: conda install -q -y numpy scipy seaborn networkx pandas
+  - cmd: conda install -q -y pytest pytables six sympy pyyaml
+  - cmd: conda install -q -y psutil numexpr nodejs ipywidgets bokeh
+  - cmd: conda install -q -y numba
```
Comment (herbertludowieg) [inline appveyor.yml]: Seems to take care of versions to be newer in some cases. Not sure why conda-forge is doing weird things all of a sudden.

File: exatomic/interfaces/tests/test_cube.py (hunk 1 above)
Comment (avmarchenko) [reply exatomic/interfaces/tests/test_cube.py]: Might be able to simplify this to `if "win" not in platform.casefold()`.

Comment (herbertludowieg) [reply exatomic/interfaces/tests/test_cube.py]: Ah yes, that would be better.

**Comment by herbertludowieg** [comment]
What should we do about the failing appveyor tests? It seems to come more from numba.

**Pull request merged**